code/modules/projectiles/ammo_datums.dm

```diff
@@ -1184,6 +1245,12 @@ datum/ammo/bullet/revolver/tp44
 	shrapnel_chance = 25
 	sundering = 2.5
 
+/datum/ammo/bullet/minigun/mech
+	name = "40mm minigun bullet"
```

Comment:
```suggestion
	name = "vulcan bullet"
```
tbh, i don't think we'd call it a 'mini' gun since it's upscaled to 20mm (40mm, no less)